File: passbook-management-api/src/main/java/com/uit/passbook_management_api/entity/QuyDinh.java
package com.uit.passbook_management_api.entity;

import jakarta.persistence.*;
import java.math.BigDecimal;

@Entity
@Table(name = "quy_dinh")
public class QuyDinh {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    @Column(name = "tien_gui_toi_thieu")
    private BigDecimal tienGuiToiThieu;

    @Column(name = "tien_gui_them_toi_thieu")
    private BigDecimal tienGuiThemToiThieu;

    @Column(name = "thoi_gian_gui_toi_thieu_ngay")
    private Integer thoiGianGuiToiThieuNgay;

    @Column(name = "lai_suat_kkh")
    private BigDecimal laiSuatKkh;

    @Column(name = "lai_suat_3t")
    private BigDecimal laiSuat3t;

    @Column(name = "lai_suat_6t")
    private BigDecimal laiSuat6t;

    // TODO: Thêm Getters / Setters
}

File: frontend-csharp/Models/QuyDinhModel/QuyDinh.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Text.Json.Serialization;

namespace frontend_csharp.Models.QuyDinhModel
{
    public class QuyDinh
    {
        [JsonPropertyName("id")]
        public int Id { get; set; }

        [JsonPropertyName("tienGuiToiThieu")]
        public decimal TienGuiToiThieu { get; set; }

        [JsonPropertyName("tienGuiThemToiThieu")]
        public decimal TienGuiThemToiThieu { get; set; }

        [JsonPropertyName("thoiGianGuiToiThieuNgay")]
        public int ThoiGianGuiToiThieuNgay { get; set; }

        [JsonPropertyName("laiSuatKkh")]
        public decimal LaiSuatKkh { get; set; }

        [JsonPropertyName("laiSuat3t")]
        public decimal LaiSuat3t { get; set; }

        [JsonPropertyName("laiSuat6t")]
        public decimal LaiSuat6t { get; set; }
    }
}